AngularLanguageService/packages.config

```diff
@@ -1,154 +1,158 @@
+  <package id="System.Private.Uri" version="4.3.2" targetFramework="net472" />
+  <package id="System.Reflection" version="4.3.0" targetFramework="net472" />
+  <package id="System.Reflection.Emit" version="4.6.0" targetFramework="net472" />
+  <package id="System.Reflection.Emit.Lightweight" version="4.6.0" targetFramework="net472" />
+  <package id="System.Reflection.Extensions" version="4.3.0" targetFramework="net472" />
+  <package id="System.Reflection.Primitives" version="4.3.0" targetFramework="net472" />
+  <package id="System.Resources.ResourceManager" version="4.3.0" targetFramework="net472" />
+  <package id="System.Runtime" version="4.3.0" targetFramework="net472" />
+  <package id="System.Runtime.CompilerServices.Unsafe" version="4.6.0" targetFramework="net472" />
+  <package id="System.Runtime.Extensions" version="4.3.0" targetFramework="net472" />
+  <package id="System.Runtime.Handles" version="4.3.0" targetFramework="net472" />
+  <package id="System.Runtime.InteropServices" version="4.3.0" targetFramework="net472" />
+  <package id="System.Runtime.InteropServices.RuntimeInformation" version="4.3.0" targetFramework="net472" />
+  <package id="System.Runtime.Numerics" version="4.3.0" targetFramework="net472" />
+  <package id="System.Security.AccessControl" version="4.6.0" targetFramework="net472" />
+  <package id="System.Security.Cryptography.Algorithms" version="4.3.0" targetFramework="net472" />
+  <package id="System.Security.Cryptography.Encoding" version="4.3.0" targetFramework="net472" />
+  <package id="System.Security.Cryptography.Primitives" version="4.3.0" targetFramework="net472" />
+  <package id="System.Security.Cryptography.X509Certificates" version="4.3.0" targetFramework="net472" />
+  <package id="System.Security.Principal.Windows" version="4.6.0" targetFramework="net472" />
+  <package id="System.Text.Encoding" version="4.3.0" targetFramework="net472" />
+  <package id="System.Text.Encoding.Extensions" version="4.3.0" targetFramework="net472" />
+  <package id="System.Text.RegularExpressions" version="4.3.0" targetFramework="net472" />
+  <package id="System.Threading" version="4.3.0" targetFramework="net472" />
+  <package id="System.Threading.Tasks" version="4.3.0" targetFramework="net472" />
+  <package id="System.Threading.Tasks.Dataflow" version="4.9.0" targetFramework="net472" />
+  <package id="System.Threading.Tasks.Extensions" version="4.5.3" targetFramework="net472" />
+  <package id="System.Threading.Timer" version="4.3.0" targetFramework="net472" />
+  <package id="System.ValueTuple" version="4.5.0" targetFramework="net472" />
+  <package id="System.Xml.ReaderWriter" version="4.3.0" targetFramework="net472" />
+  <package id="System.Xml.XDocument" version="4.3.0" targetFramework="net472" />
+  <package id="VSLangProj" version="7.0.3301" targetFramework="net472" />
+  <package id="VS.ExternalApis.WebTools.References" version="16.9.673" targetFramework="net472" />

```

Comment:
We should have a nuget config so we have a sense of what package location dependencies we have. This will help manage them in the future and also makes it easier to develop locally. I didn't have the source for `VS.ExternalApis.WebTools.References` in my global nuget.config so I had to add it manually, but having a nuget.config in this repo would be more consistent.

Reply:
Added a nuget config. Since the nuget restore task in the pipeline now has a check that doesn't allow more than one feed, I created a new feed with two upstreams [here](https://devdiv.visualstudio.com/DevDiv/_packaging?_a=feed&feed=AngularLanguageService-aggregate). It should be set to the correct access? Let me know if not.